apps/mosaico/src/Cache/Pubsub.purs

```diff
@@ -0,0 +1,30 @@
+module Mosaico.Cache.Pubsub where
+
+import Prelude
+
+import Control.Promise (Promise, toAffE)
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Data.Maybe (Maybe)
+import Data.Nullable (Nullable, toMaybe)
+import Lettera.Models (CategoryLabel(..))
+import Lettera (LetteraResponse(..))
+
+foreign import data Message :: Type
+
+foreign import subscribeImpl :: (Message -> Effect Unit) -> Effect (Promise Unit)
```

Comment:
Is it a deliberate decision not to use `Effect.Uncurried` and `Data.Function.Uncurried` with these?

Reply:
All the foreign functions in this module use one parameter so it wouldn't really add anything. I don't know why the module even has `runEffectFn1` since it's pretty much a no op.